third_party/3/pkg_resources/__init__.pyi

```diff
@@ -202,7 +202,19 @@ class IMetadataProvider:
 
 class ResolutionError(Exception): ...
 class DistributionNotFound(ResolutionError): ...
-class VersionConflict(ResolutionError): ...
+
+class VersionConflict(ResolutionError):
+    @property
+    def dist(self) -> List[Any]: ...
+    @property
+    def req(self) -> List[Any]: ...
+    def report(self) -> str: ...
+    def with_context(self, required_by: Dict[str, Any]) -> ContextualVersionConflict: ...
+
+class ContextualVersionConflict(VersionConflict):
+    @property
+    def required_by(self) -> List[Any]: ...
```

Comment:
This as well. As far as I can see, it gets passed `Set`.

Reply:
I am kinda confused, but fixed!